src/main/resources/assets/securitycraft/lang/fr_fr.json

```diff
@@ -0,0 +1,767 @@
+	"messages.securitycraft:module.manager": "Gestionnaire de Module",
+	"messages.securitycraft:module.whitelisted": "Vous êtes sur la liste blanche.",
+	"messages.securitycraft:module.blacklisted": "Vous êtes sur la liste noire.",
+	"messages.securitycraft:module.saved": "Données enregistrées.",
+	"messages.securitycraft:module.notHoldingForData": "Vous devez tenir le module avec lequel vous souhaitez enregistrer les données.",
+	"messages.securitycraft:module.nothingSaved": "Aucune donnée de module n’est enregistrée.",
+	"messages.securitycraft:module.alreadyContained": "Le module que vous tenez contient déjà le joueur #.",
+	"messages.securitycraft:module.added": "Ajout de # sur le module tenu.",
+	"messages.securitycraft:module.removed": "Suppression de # du module tenu.",
+	"messages.securitycraft:module.notHoldingForModify": "Vous devez tenir le module que vous souhaitez modifier !",
+	"messages.securitycraft:notOwned": "Je suis désolé, vous ne pouvez pas faire ça sur ce bloc. Ce bloc appartient à #.",
+	"messages.securitycraft:adminTool.owner.name": "Propriétaire: #",
+	"messages.securitycraft:adminTool.owner.uuid": "Propriétaire UUID: #",
+	"messages.securitycraft:adminTool.password": "Mot de passe: #",
+	"messages.securitycraft:adminTool.equippedModules": "Modules:",
+	"messages.securitycraft:adminTool.noInfo": "Ce bloc n’a aucune information à afficher!",
+	"messages.securitycraft:reinforcedSlab": "Dalle blindée",
+	"messages.securitycraft:reinforcedSlab.cannotDoubleSlab": "Vous devez être propriétaire de ce bloc pour le transformer en double dalle.",
+	"messages.securitycraft:cameraMonitor.cannotView": "Vous ne pouvez pas voir une caméra qui ne vous appartient pas.",
+	"messages.securitycraft:cameraMonitor.bound": "Caméra associée à # au moniteur.",
+	"messages.securitycraft:cameraMonitor.unbound": "Caméra dissociée à # du moniteur.",
+	"messages.securitycraft:cameraMonitor.rightclickToView": "Faites un clic droit sur une caméra de sécurité pour la voir.",
+	"messages.securitycraft:mrat.noSlots": "Il n’y a plus d’emplacements vides pour associer cette mine!",
+	"messages.securitycraft:mrat.cantBind": "Vous ne pouvez pas associer une mine qui ne vous appartient pas.",
+	"messages.securitycraft:mrat.bound": "Association de la mine à # à l’outil d’accès à distance.",
+	"messages.securitycraft:mrat.unbound": "Dissociation de la mine à # de l’outil d’accès à distance.",
+	"messages.securitycraft:srat.noSlots": "Il n’y a plus d’emplacements vides pour associer cette sentinelle !",
+	"messages.securitycraft:srat.cantBind": "Vous ne pouvez pas associer une sentinelle qui ne vous appartient pas.",
+	"messages.securitycraft:srat.bound": "Association de la sentinelle à # à l’outil d’accès à distance.",
+	"messages.securitycraft:srat.unbound": "issociation de la sentinelle à # de l’outil d’accès à distance.",
+	"messages.securitycraft:universalBlocModifier.cantModify": "\"",
```

Comment:
You accidentally removed the `k` from `Block` in the language key here. It should be `messages.securitycraft:universalBlockModifier.cantModify`.

Reply:
Still needs to be changed.